coursebuilder/modules/courses/lessons.py

```diff
@@ -372,6 +372,7 @@ def _apply_gcb_tags(self, text):
         return jinja_utils.get_gcb_tags_filter(self)(text)
 
     def _show_all_contents(self, student, unit, student_view):
+        self.template_value['unit_id'] = unit.unit_id
```

Comment:
@Tawakalt we have recently discovered that it is evil to make changes to the GCB core. This looks like a property that you can get in the template.

Reply:
@chegejames I couldn't get the property I needed from the template, that was why I initially used the title property. But I can look deeper to see if I'll find what I need.

Reply:
@chegejames I have worked on this.

Reply:
@chegejames No, I'm not. Check the most recent commit.

Reply:
That was a pending comment from last time. Sorry about that.